kfs-core/src/main/resources/edu/arizona/kfs/vnd/businessobject/datadictionary/VendorDetail.xml

```diff
@@ -27,5 +27,21 @@
 	<bean id="VendorDetail-vendorParentName" parent="VendorDetail-vendorParentName-parentBean">		
 		<property name="maxLength" value="40" />		
 	</bean>
+	
+	<bean id="VendorDetail-lookupDefinition" parent="VendorDetail-lookupDefinition-parentBean">
+		<property name="resultFields">
+			<list>
```

Comment:
Looks like this list was copied from KFS 3.0 VendorDetail DD, not KFS 6.0 VendorDetail DD.
1. vendorAliases attribute name should be vendorAliasesAsString.
2. bean for vendorHeader.vendorCountry.name attribute name is missing.
3. vendorCommodities attribute name should be vendorCommoditiesAsString.
4. vendorHeader.vendorSupplierDiversities attribute name should be vendorHeader.vendorSupplierDiversitiesAsString.